flytekit/bin/entrypoint.py

```diff
@@ -376,6 +376,10 @@ def _execute_task(
         dynamic_addl_distro,
         dynamic_dest_dir,
     ) as ctx:
+        import sys
+
+        if "." not in sys.path:
+            sys.path.append(".")
```

Comment:
I think it's best to be explicit here and add the fully path for the current working directory:

```suggestion
        sys.path.append(os.getcwd())
```

Although, I do not think there is harm to include the working directory twice. If you want to make sure to not add the working directory twice:

```python
working_dir = os.getcwd()
if all(os.path.realpath(path) != working_dir for path in sys.path):
    sys.path.append(working_dir)
```

Reply:
Yes agreed.  This seems safer than adding `.`.  I'm worried about users who use both `.relative` imports as well as absolute imports.  We've seen weird cases of repeat-importing of modules before.

Reply:
Got it. Thank you for your advice.